op-node/p2p/rpc_api.go

```diff
@@ -20,8 +20,8 @@ type PeerInfo struct {
 	ENR             string   `json:"ENR"`       // might not always be known, e.g. if the peer connected us instead of us discovering them
 	Addresses       []string `json:"addresses"` // multi-addresses. may be mix of LAN / docker / external IPs. All of them are communicated.
 	Protocols       []string `json:"protocols"` // negotiated protocols list
-	//GossipScore float64
-	//PeerScore float64
+	// GossipScore float64
+	// PeerScore float64
```

Comment:
FWIW Generally better practice to split smaller style changes out into a separate pr so as to not bloat the pr diff, but no big deal this time around

Reply:
Totally, I thought about not doing the style changes as well, but kept it due to the small diff size.

Will keep this in mind for future PRs if there's large formatting changes

Reply:
Awesome thank you 👍